app.ts

```diff
@@ -0,0 +1,143 @@
+//Loads the table to the screen
+window.onload = () => GetColumnNames();
+
+//Variable declarations
+let totalRecordCount;
+let columnNames: string[];
+let records: any[];
+let from: number = 1;
+let to: number = 30;
+
+//Function to retrieve total amount of records
+function GetNumberOfRecords() {
+  $.ajax({
+    url: "/recordCount",
+    type: "GET",
+    timeout: 12000,
+    dataType: "text"
+  })
+    .done(function(responseText: any) {
+      totalRecordCount = responseText;
+    })
+    .fail(function() {
```

Comment:
I would rather you use the arrow function here. That looks something like 
```ts
    .fail(() => {
```
For your purpose at the moment, this does not do anything really - so I'm not going to explain the significance. 
Just know that at IMQS, we prefer these arrow functions.

This request to change most of these to arrow functions applies to basically every internally declared function, and I will comment on it just now.